### PR TITLE
Update data for ASU (FD #7305)

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

**Comment by matyasselmeci** [comment]
I set the VOOwnership for the resource to be 100% Other; I don't know if that's actually what we want.

**Comment by brianhlin** [comment]
I believe we set `OSG: 100` for purely opportunistic sites, though honestly that's pretty non-sensical.

**Comment by matyasselmeci** [comment]
OK, I updated the VOOwnership and fixed all the CI errors; ready for review.

File: topology/Arizona State University/ASU/DELL_M420_ASU.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Disable: false
+Production: true
+SupportCenter: Self Supported
+GroupID: 486
+GroupDescription: Arizona State University Research Computing.
+Resources:
+  DELL_M420_ASU:
```
Comment (brianhlin) [inline topology/Arizona State University/ASU/DELL_M420_ASU.yaml]: @zvada requested that we change this to `OSG_DELL_M420_ASU`

File: projects/asurcosg.yaml [hunk 2]
```diff
@@ -4,8 +4,8 @@ FieldOfScience: Computer and Information Science and Engineering
...
 Organization: Arizona State University
-PIName: Jay Etchings
+PIName: Johnathan Lee
 Sponsor:
-  CampusGrid:
-    ID: '14'
-    Name: OSG Connect
+  VirtualOrganization:
+    ID: 30
+    Name: OSG
```
Comment (brianhlin) [inline projects/asurcosg.yaml]: Let's not make any changes to the project for now

File: topology/Arizona State University/ASU/DELL_M420_ASU.yaml (hunk 1 above)
Comment (matyasselmeci) [reply topology/Arizona State University/ASU/DELL_M420_ASU.yaml]: OK sure

File: projects/asurcosg.yaml (hunk 2 above)
Comment (matyasselmeci) [reply projects/asurcosg.yaml]: OK

**Comment by matyasselmeci** [comment]
"Research Computing" is too generic of a site name; most site names also have the facility name in there somewhere, How about "ASU Research Computing"?

**Comment by matyasselmeci** [comment]
Ready for re-review.

**Pull request merged**